dev/set_matrix.py

```diff
@@ -441,7 +442,7 @@ def main(args):
 
     if "GITHUB_ACTIONS" in os.environ:
         set_action_output("matrix", json.dumps(matrix, cls=CustomEncoder))
-        set_action_output("is_matrix_empty", "true" if len(matrix) == 0 else "false")
```

Comment:
`len(matrix)` always returns 2 because it looks like `{"include": ..., "job_name": ...]}`